deployer/deployer-services/src/main/java/io/hyscale/deployer/services/deployer/impl/KubernetesDeployer.java

```diff
@@ -151,6 +139,11 @@ public ResourceStatus status(String namespace, Manifest manifest, AuthConfig aut
         }
     }
 
+    @Override
+    public List<Pod> getPods(String namespace, String appName, String serviceName, K8sAuthorisation k8sAuthorisation) throws Exception {
```

Comment:
Is this no longer used by platform?

Reply:
They are using these changes, I'm making minor changes in this model. Will push it as a next commit.